### ISSUE TITLE
`tinytable` S4 refactor: output formats

### ISSUE THREAD

**Issue by vincentarelbundock** [strengejacke/ggeffects#462]
FYI, I just merged a refactor of `tinytable` to `main` on Github. The big change is that the object is now an S4 class. This will make it much easier for you to modify the default printed output format. With the dev version (>0.0.5), you can just do this:

```{r}
library(tinytable)
x <- tt(mtcars[1:4, 1:4])
x@output <- "html"
x
```

**Comment by strengejacke** [comment]
Ah, nice! Will this be a breaking change, or will

```r
  m <- attr(out, "tinytable_meta")
  m$output <- "html"
  attr(out, "tinytable_meta") <- m
``` 

Also work for tt > 0.05?

**Comment by vincentarelbundock** [comment]
yeah, unfortunately this has to be a breaking change, sorry! (the joys of 0.0.X releases ;))

**Comment by vincentarelbundock** [comment]
I'll let you know when I'm preparing to release to CRAN

**Comment by vincentarelbundock** [comment]
thanks again for your indulgence. I'm hoping things will be more stable from now on.

**Comment by strengejacke** [comment]
No problem ;-)
